tools/ci/src/main.rs

```diff
@@ -225,19 +229,28 @@ fn main() {
             "--workspace",
             "--all-features",
             "--no-deps",
-            "--document-private-items",
+            "-Zunstable-options",
+            "-Zrustdoc-scrape-examples",
         ];
         if flags.contains(Flag::KEEP_GOING) {
             args.push("--keep-going");
         }
+        if flags.contains(Flag::DEPLOY_DOCS) {
+            args.remove(args.iter().position(|&arg| arg == "--workspace").unwrap());
+            args.push("-p");
+            args.push("bevy");
+        }
 
         test_suite.insert(
             Check::DOC_CHECK,
             vec![CITest {
-                command: cmd!(sh, "cargo doc {args...}"),
+                command: cmd!(sh, "cargo +nightly doc {args...}"),
```

Comment:
it's a pain to require nightly in the tools. if it's launch using stable, it should keep on stable and not stealthily use a nightly

it's not the first place it's used in this script, I would rather that this would be fixed rather than add more places where this appear

Reply:
Nightly is needed if you want to check correctly the docs as in docs.rs
I change it so it will only be use if you use the flag `DEPLOY_DOCS`